### ISSUE TITLE
[COGNITION]: Alert Boxes - Icons SHOULD be hidden from screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#215]
@1Copenut commented on [Thu Jun 18 2020](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10463)

# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Our [Alert Box component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/) uses icons on the left, that are appearing as empty tab stops for JAWS. These do not announce to users and could be confusing. Let's hide them using the updated snippets below.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Icons are not being announced to screen readers. JAWS in particular.
- [ ] Create and/or modify unit tests for this component to include an aXe scan

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs (optional)
![Screen Shot 2020-06-18 at 5 15 42 PM](https://user-images.githubusercontent.com/934879/85077285-88ad9500-b187-11ea-8969-0244c7d04a5b.png)

**Comment by ddzz** [comment]
Fixed in this PR: https://github.com/department-of-veterans-affairs/component-library/pull/74

**Comment by bkjohnson** [comment]
Closing since it's been fixed. Feel free to reopen if that isn't the case.

**Comment by bkjohnson** [comment]
One caveat - the fix appears to be made to the `<va-alert>` Web Component, so the `<AlertBox>` React component could still have it. If that's the case, applications can upgrade to the Web Component in order to resolve the issue.
